packages/snaps-utils/src/manifest/manifest.ts

```diff
@@ -259,15 +261,21 @@ export async function runFixes(
 
     fixResults = await runValidators(fixResults.files, rules);
     shouldRunFixes = hasFixes(fixResults, errorsOnly);
+
+    mergedReports.push(
+      ...fixResults.reports.filter(
```

Comment:
Theoretically this can overflow the stack, unsure if we care for this specific logic though

Reply:
Can rewrite to use `.forEach` if you prefer.

Reply:
Up to you